src/dscanner/main.d

```diff
@@ -376,6 +376,8 @@ Options:
     --errorFormat|f <pattern>
         Format errors produced by the style/syntax checkers. The default
         value for the pattern is: "%2$s".
```

Comment:
Maybe I'm missing something, but shouldn't this text say that the default value for the pattern is:
`"{filepath}({line}:{column})[{type}]: {message}"`?